README.md

```diff
@@ -1,10 +1,21 @@
 
 # sbt-accessibility-linter
 
-This is a placeholder for an `sbt` plugin to identify accessibility issues in HTML markup.
+This is an `sbt` plugin that can be used to identify accessibility issues in your HTML markup.
+
+It's inspired by [jest-axe](https://github.com/nickcolley/jest-axe) from the Node.js ecosystem, but designed to
+work with Scala frontend microservices. It also combines Axe checks with markup validation checks from VNU.
+
+:warning: **Important**: This tool does not guarantee what you build is accessible.
+
+`sbt-accessibility-linter` is a linter, it can identify common issues but cannot guarantee that your service will work 
+for all users.
```

Comment:
@matthewmascord maybe we could link to https://github.com/hmrc/accessibility and say something like "it can help you check the accessibility of your service before you get an audit"